apps/server/src/modules/provisioning/strategy/unknown/unknown.strategy.ts

```diff
@@ -0,0 +1,18 @@
+import { ProvisioningStrategy } from '@src/modules/provisioning/strategy/base.strategy';
+import { UnknownResponse } from '@src/modules/provisioning/strategy/unknown/unknown.response';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class UnknownProvisioningStrategy extends ProvisioningStrategy<UnknownResponse> {
+	override getProvisioningData(): Promise<UnknownResponse> {
+		return Promise.resolve(
```

Comment:
+1, just for the sake of readability. This could become the 'default', i.e. undefined strategy, which, by default, fails. In the later sprint you may still take a copy of this 'undefined' strategy?

Reply:
No there is no default strategy. This is only a placeholder.

Reply:
for me, even a placeholder can return an error temporarily.
Since we aim for PRs to live as short as possible, that makes sure the behavior is always consistent even when the "placeholder" code is merged.

Reply:
We throw now an NotImplementedError